docs/source/getting_started/getting_started.md

```diff
@@ -237,20 +194,17 @@ curl -X POST "$URL/semantic_search" \
     --header "authorizationToken: $FRONTOFFICE_TOKEN" \
     --header "Content-Type: application/json" \
     --data '{
-        "document": { "id": { "document_id": "xayn_cd5604c", "sub_id": 2 } },
+        "document": { "id": "xayn_cd5604c" },
         "include_properties": true
     }'
 ```
 
-The result contains a list of snippets that are similar to the identified snippet.
-
-If only documents with a single snippet are used you can provide only the document id
-like this: `"document": { "id": "xayn_cd5604c" },`.
+The result contains a list of snippets similar to the identified document.
 
-Be aware that at this moment, this is equivalent to doing a semantic search on the first snippet of a document.
-In the future, this behaviour could change to better represent the intent to search for something similar to the whole document. If you want to explicitly search for the first snippet use a _snippet id_.
+Please note that when using [document with multiple snippets](#documents-with-multiple-snippets) it is recommended to use the snippet id instead of an document id. More details can be found in the chapter
+about documents with multiple snippets and the API documentation.
```

Comment:
```suggestion
Please note that when using [document with multiple snippets](#documents-with-multiple-snippets), it is recommended to use the snippet id instead of a document id. More details can be found in the chapter
about documents with multiple snippets and the API documentation.
```